README.md

```diff
@@ -120,23 +121,15 @@ Have you had a good experience with the **Solr Operator**? Why not share some lo
 
 We welcome issue reports [here](../../issues); be sure to choose the proper issue template for your issue, so that we can be sure you're providing the necessary information.
 
-Before sending a [Pull Request](../../pulls), please make sure you read our
-[Contribution Guidelines](https://github.com/bloomberg/.github/blob/master/CONTRIBUTING.md).
-
 ## License
 
 Please read the [LICENSE](LICENSE) file here.
 
-## Code of Conduct
```

Comment:
Is there no Apache code of conduct we can use here instead?

Reply:
What about this? https://www.apache.org/foundation/policies/conduct

Reply:
I was trying to look up something that I thought we used for PR creation in the `lucene-solr` project, just to reuse and be consistent but if I'm unable to find it, this will work :)

Reply:
@HoustonPutman - we shouldn't add any non-ASF communication channels here as they aren't official.